tests/bucket_test.cpp

```diff
@@ -10,4 +10,14 @@ TEST_SUITE("Bucket") {
         REQUIRE(b.find(4, &v));
         REQUIRE(v == 5);
     }
+    TEST_CASE_FIXTURE(DiskManagerFixture, "Insert/Delete") {
+        Bucket<int, int> b(&dm);
+        b.insert(1, 2);
+        b.insert(2, 3);
+        b.remove(1);
+        int v;
+        REQUIRE(!b.find(1, &v));
+        REQUIRE(!b.isempty());
```

Comment:
Misspelled method name. Won't even compile.
Please add proper test cases for is_empty and is_full